src/SdkCommon/AppAuthentication/Azure.Services.AppAuthentication.TestCommon/Microsoft.Azure.Services.AppAuthentication.TestCommon.csproj

```diff
@@ -0,0 +1,61 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  
+	<PropertyGroup>
+    
+		<TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
+  
+		<SignAssembly>True</SignAssembly>
```

Comment:
@varunsh-msft if you model your project file after one of the existing project file, that will help eliminating redundant properties.